src/renderer/components/misc/MarkDown.vue

```diff
@@ -26,7 +26,6 @@ marked.use({
 const allowedTags = ["p", "em", "b", "strong", "ul", "ol", "li", "code", "pre", "blockquote", "span", "del", "body"];
 
 const allowedAttributes: string[] = [];
-
 if (props.allowLinks) {
     allowedTags.push("a");
 
```

Comment:
I tried sanitizing the link. Till I realized that it does not help at all since all the other stuff is not sanitized. And all the other functions no not sanitize their stuff...

Reply:
With this will-navigate is the only line of defense